guides/doc-Configuring_virt_who_VM_Subscriptions/master.adoc

```diff
@@ -6,6 +6,31 @@ include::common/header.adoc[]
 
 = {ConfiguringVMSubscriptionsDocTitle}
 
+////
+DOC INFO POC
+
+Category: Managing hosts
+
+Published: Satellite, Orcharhino
+
+Persona: System administrator (probably corresponds to 'Admin' in Contributing.md)
+
+Goal: Install and configure the "virt-who" plugin to enable accurate tracking of virtual machine subscriptions on supported hypervisors.
+"virt-who" maps virtual guests to physical hosts and reports the data to Project Server.
```

Comment:
https://github.com/theforeman/foreman-documentation/issues/4494 says that "goal" corresponds to "job to be done". Our downstream job to be done definition should not be based on specific features, products, or solutions. In this case, that specific feature/solution is virt-who.

To resolve this, I suggest to either re-define what the 'goal' of a guide is (so that we don't create misleading "job to be done" descriptions), or to rewrite these lines to focus on what users want to achieve + desired outcome, rather than how they will achieve it.